File: win32/xampp/contrib/mysql.php
<html>
<head>
<title>PHP und MySQL</title>
</head>
<body>
<h1>PHP und MySQL</h1>
<table border="1">
<tr>
    <th>Vorname</th>
    <th>Nachname</th>
    <th>Telefonnummer</th>
</tr>
<?php
    mysql_connect("localhost", "oswald", "geheim");
    mysql_select_db("phonebook");

    $query = "SELECT * FROM users";
    $result = mysql_query($query);
    while ($row = mysql_fetch_array  ($result))
    {
        echo "<tr>";
        echo "<td>".$row['firstname']."</td>";
        echo "<td>".$row['lastname']."</td>";
        echo "<td>".$row['phone']."</td>";
        echo "</tr>";
    }

?>
</table>
</body>
</html>

File: contrib/mysql.pl
#!/opt/lampp/bin/perl
use DBI;

print "Content-Type: text/html\n\n";

my $dsn="dbi:mysql:phonebook:localhost";
my $dbh=DBI->connect("$dsn","oswald","geheim") or
        die "Kann Datenbank nicht erreichen!";

print "<html>";
print "<head>";
print "<title>Perl und MySQL</title>";
print "</head>";
print "<body>";
print "<h1>Perl und MySQL</h1>";
print "<table border=\"1\">";
print "<tr>";
print "<th>Vorname</th>";
print "<th>Nachname</th>";
print "<th>Telefonnummer</th>";
print "</tr>";

my $query="SELECT * FROM users";

my $prep_sql=$dbh->prepare($query) or  die print "Can't prepare";
$prep_sql->execute() or die print "Can't  execute";

while (my @row = $prep_sql->fetchrow_array())
{
    print "<tr>";
    print "<td>".$row[1]."</td>";
    print "<td>".$row[2]."</td>";
    print "<td>".$row[3]."</td>";
    print "</tr>";
}

$prep_sql->finish();
$dbh->disconnect();

print "</table>";
print "</body>";
print "</html>";
